modules/sdk-core/src/account-lib/mpc/tss/ecdsa/types.ts

```diff
@@ -164,13 +164,14 @@ export interface SignCombineRT {
 
 export interface Signature {
   y: string;
+  recid: number;
   r: string;
   s: string;
 }
 
 export interface SignRT {
```

Comment:
NIT: We can remove this SIgnRT from everywhere as it is equivalent to SShare.

Reply:
or may be for backward compatibility
```suggestion
export interface SignRT = SShare;
```

Reply:
Done